src/Microsoft.Win32.Primitives/src/Properties/AssemblyInfo.cs

```diff
@@ -26,5 +26,4 @@
 [assembly: NeutralResourcesLanguageAttribute("en-US")]
 
 [assembly: CLSCompliant(true)]
-[assembly: ComVisible(false)]
```

Comment:
Note this is going to conflict with #771/#772 with the mirroring so please do not commit any changes to AssemblyInfo.cs files. If we need ComVisible(false) on everything we should include it in the generated _AssemblyInfo.cs file we are going to be including soon.